common/src/main/java/com/wynntils/models/stats/builders/MiscStatKind.java

```diff
@@ -44,30 +45,50 @@ public enum MiscStatKind {
     // (The last three are currently only found on crafted gear)
 
     // Special tome only stats, which are only found as base stats on tomes
-    SLAYING_XP("Slaying XP", StatUnit.PERCENT, "slayingXP", "SLAYING_XP"),
-    GATHERING_XP("Gathering XP", StatUnit.PERCENT, "gatheringXP", "GATHERING_XP"),
-    DUNGEON_XP("Dungeon XP", StatUnit.PERCENT, "dungeonXP", "DUNGEON_XP");
+    SLAYING_XP("Slaying XP", StatUnit.PERCENT, "slayingXP", "SLAYING_XP", StatType.SpecialStatType.TOME_BASE_STAT),
+    GATHERING_XP(
+            "Gathering XP", StatUnit.PERCENT, "gatheringXP", "GATHERING_XP", StatType.SpecialStatType.TOME_BASE_STAT),
+    DUNGEON_XP("Dungeon XP", StatUnit.PERCENT, "dungeonXP", "DUNGEON_XP", StatType.SpecialStatType.TOME_BASE_STAT),
+
+    // Charm stats
+    LEVELED_XP_BONUS(
+            "XP from Lv. ${} content",
```

Comment:
So this is more like a "stat template", where you can have different instances with specific levels? That really messes up our system, doesn't it? We'l need to store the level as part of the stat "key", not as the stat value...

Reply:
It only had to be patched in StatModel for getting the display name from string and for getting a stat from display name. Although it is kind of hacky, I am not sure I can see a more obvious way forward atm.